spec/Subscriber/CategoryViewSpec.php

```diff
@@ -88,7 +88,18 @@ public function it_will_encode_category_path_correctly(
         NavigationPageLoadedEvent $event
     ) {
         $categoryEntity->getCustomFields()->willReturn([]);
-        $extension->assign(Argument::withEntry('communication', Argument::withEntry('add-params', Argument::containingString('filter=CategoryPath%2C+%5B%5D%3ABooks%2520%2526%2520Sports%2FHome%2520%2526%2520Garden'))))->shouldBeCalled();
+        $extension->assign(Argument::withEntry('communication', Argument::withEntry('category-page', Argument::containingString('filter=CategoryPath%2C+%5B%5D%3ABooks%2520%2526%2520Sports%2FHome%2520%2526%2520Garden'))))->shouldBeCalled();
+        $this->onPageLoaded($event);
+    }
+
+    public function it_will_not_add_category_path_to_add_params(
+        CategoryEntity $categoryEntity,
+        Struct $extension,
+        NavigationPageLoadedEvent $event
+    )
+    {
+        $categoryEntity->getCustomFields()->willReturn([]);
+        $extension->assign(Argument::withEntry('communication', Argument::withEntry('category-page', Argument::containingString('filter=CategoryPath%2C+%5B%5D%3ABooks%2520%2526%2520Sports%2FHome%2520%2526%2520Garden'))))->shouldBeCalled();
```

Comment:
This test is not testing what is contained in its name.
With current state it would be 'it_will_add_category_path_to_category_page`,

It's good to test opposite conditions so if we check if `category-page` contains specific value, let's test also if `add-params` does not contains them

Reply:
added additional tests